sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/README.md

```diff
@@ -20,6 +20,12 @@ Install the Azure Monitor Exporter for OpenTelemetry .NET with [NuGet](https://w
 dotnet add package Azure.Monitor.OpenTelemetry.Exporter --prerelease
 ```
 
+#### Nightly builds
```

Comment:
is this specific to AzureMonitor? if not, link to someplace common where this is docoumented.

Reply:
I agree, but I don't see this referenced anywhere else in the repo. 
We should discuss this with Azure SDK team.

Reply:
I found the information. Azure calls it their dev feed. Updating this PR now!